backend/service/k8s/clientset.go

```diff
@@ -55,6 +56,13 @@ func newClientsetManager(rules *clientcmd.ClientConfigLoadingRules) (ClientsetMa
 			return nil, fmt.Errorf("could not load restconfig: %w", err)
 		}
 
+		if restConfig == nil {
+			restConfig, err = rest.InClusterConfig()
+			if err != nil {
+				return nil, fmt.Errorf("could not load InClusterConfig: %w", err)
+			}
+		}
```

Comment:
pretty sure this needs to happen at the beginning. i think this is already in the flow of reading the file?

Reply:
yeah this is not correct, was just a little hard to debug in docker working on the correct fix now.

Reply:
i think after loop if `lookup` is empty then do inclusterconfig